spring-integration-jmx/src/test/java/org/springframework/integration/monitor/MonitorTests.java

```diff
@@ -87,7 +87,9 @@ public void testStats() throws InterruptedException {
 			Object result = invocation.callRealMethod();
 			afterSendLatch.countDown();
 			return result;
-		}).when(channelMetrics).afterSend(Mockito.any(MetricsContext.class), Mockito.eq(Boolean.TRUE));
+		}).when(channelMetrics)
+			.afterSend(Mockito.any(org.springframework.integration.support.management.MetricsContext.class),
+					Mockito.eq(Boolean.TRUE));
```

Comment:
I wonder if we don't need this test any more since `MetricsContext` is deprecated...